code/game/objects/items/holosign_creator.dm

```diff
@@ -90,6 +90,51 @@
 	creation_time = 0
 	max_signs = 3
 
+/obj/item/holosign_creator/powered_shielding
+	name = "Power shield projector"
+	desc = "A holographic projector that creates a Power shield that stops the spread of heat and atmosphere changes"
+	icon_state = "signmaker_powersh"
+	holosign_type = /obj/machinery/holosign/barrier/power_shield/wall
+	creation_time = 0
+	///25 so that you can make a 5x5 room with walls and floors
+	max_signs = 25
```

Comment:
No need to document redefined variables.

Reply:
Uh I meant to just comment but it seems that another / slipped in, thanks!